config/initializers/datadog-tracer.rb

```diff
@@ -1,6 +1,8 @@
-Datadog.configure do |c|
-  c.use :rails, service_name: 'dodona'
-  c.use :delayed_job
-  c.analytics_enabled = true
-  c.version = Dodona::Application::VERSION
+if Rails.env.production? || Rails.env.staging?
+  Datadog.configure do |c|
+    c.use :rails, service_name: 'dodona'
+    c.use :delayed_job
```

Comment:
Too lazy to make a separate PR for this. It enables dalli (Memcached) tracing. Right now the rails cache traces also include the MySQL query time in case of a cache miss which makes it hard to monitor the performance of Memcached itself.

```suggestion
    c.use :delayed_job
    c.use :dalli
```